.gitignore

```diff
@@ -11,3 +11,7 @@ conan-cache/
 
 # User spesific settings
 CMakeUserPresets.json
+
+# misc
+.idea/
+conanfile.txt
```

Comment:
out of curiosity: why do we need to ignore it - will it be created during the build?

Reply:
That one was a left behind for a workaround I had before this implementation in conanfile.py. 
-> removed